config/config.example.yml

```diff
@@ -54,6 +54,53 @@ db:
 ##
 #signature_server:
 
+##
+## Invidious companion is an external program
+## for loading the video streams from YouTube servers.
+##
+## When this setting is commented out, Invidious companion is not used.
+## Otherwise, Invidious will proxy the requests to Invidious companion.
+## 
+## Note: multiple URL can be configured. In this case, invidious will
+## randomly pick one every time video data needs to be retrieved. This
+## URL is then kept in the video metadata cache to allow video playback
+## to work. Once said cache has expired, requesting that video's data
+## again will cause a new companion URL to be picked.
+##
+## The parameter private_url needs to be configured for the internal
+## communication between the companion and Invidious.
+## And public_url is the public URL from which companion is listening
+## to the requests from the user(s).
+##
+## If you are using a reverse proxy then you will probably need to
+## configure the public_url to be the same as the domain used for Invidious.
+## Also apply when used from an external IP address (without a domain).
+## Examples: https://MYINVIDIOUSDOMAIN or http://192.168.1.100:8282
```

Comment:
Maybe you could mention which paths have to go through the Companion if Invidious and Companion domains are the same.

AFAIK these are:
- `/api/manifest/dash/id/*`
- `/latest_version`

In my K8S / helm template:

```yaml
...
  routes:
  - kind: Rule
    match: Host(`{{ .Values.env.domain }}`) && PathPrefix(`/`)
    services:
    - name: {{ .Values.name }}-app
      port: {{ .Values.network.app_port_http }}
  - kind: Rule
    match: Host(`{{ .Values.env.domain }}`) && (PathPrefix(`/api/manifest/dash/id`) || PathPrefix(`/latest_version`))
    services:
    - name: {{ .Values.name }}-app
      port: {{ .Values.network.companion_port_http }}
...
```

Reply:
It's in the temporary doc: https://docs.invidious.io/companion-installation/